src/components/AppConfig/AppConfig.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { useStyles2, FieldSet, Card, LoadingPlaceholder, Switch, Stack } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { IGNORE_STEPS_ANNOTATION, IGNORE_STEPS_ANNOTATION_LIST, useCheckTypes, useSkipCheckTypeStep } from 'api/api';
+
+export type JsonData = {
+  apiUrl?: string;
+};
+
+export const AppConfig = () => {
+  const s = useStyles2(getStyles);
+  const { checkTypes, isLoading, isError } = useCheckTypes();
+  const { updateIgnoreStepsAnnotation, updateCheckTypeState } = useSkipCheckTypeStep();
+
+  return (
+    <FieldSet label="Available Check Types">
+      <div className={s.description}>Enable or disable the steps for each check type.</div>
+      {isLoading && <LoadingPlaceholder text="Loading check types..." />}
+      {isError && <div>Error loading check types</div>}
+      {!isLoading && !isError && checkTypes && checkTypes.length === 0 && <div>No check types available</div>}
+      {!isLoading && !isError && checkTypes && checkTypes.length > 0 && (
```

Comment:
nit: maybe you can simplify the logic a bit
```suggestion
      {!isLoading && !isError && !checkTypes?.length && <div>No check types available</div>}
      {!isLoading && !isError && checkTypes.length && (
```